src/main/java/ch/njol/skript/expressions/ExprBannerItem.java

```diff
@@ -0,0 +1,101 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.aliases.ItemType;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import org.bukkit.Material;
+import org.bukkit.block.banner.PatternType;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+
+@Name("Banner Pattern Item")
+@Description({
+	"Gets the item from a banner pattern type.",
+	"NOTE: Not all banner patterns have an item. Currently there are:",
+	"<ul>",
+	"<li>creeper/creeper charged</li>",
+	"<li>bricks/field masoned</li>",
+	"<li>piglin/snout</li>",
+	"<li>mojang/thing</li>",
+	"<li>border/bordure indented</li>",
+	"<li>flow, flower, globe, guster, skull</li>",
+	"</ul>"
+})
+@Examples({
+	"set {_item} to creeper charged banner pattern item",
+	"set {_item} to snout banner pattern item",
+	"set {_item} to thing banner pattern item"
+})
+@Since("INSERT VERSION")
+public class ExprBannerItem extends SimpleExpression<ItemType> {
+
+	static {
+		Skript.registerExpression(ExprBannerItem.class, ItemType.class, ExpressionType.SIMPLE,
+			"%bannerpatterns% item[s]");
```

Comment:
```suggestion
			"%*bannerpatterns% item[s]");
```
literal only, this is a dangerous pattern